js/renovation/ui/common/__tests__/widget.test.tsx

```diff
@@ -544,6 +546,45 @@ describe('Widget', () => {
         });
       });
 
+      describe('windowResizeEffect', () => {
+        afterEach(() => {
+          jest.clearAllMocks();
+        });
+
+        it('should subscribe to windowResizeEffect event', () => {
```

Comment:
Very strange test name for me

Reply:
You are right. I have changed it - https://github.com/DevExpress/DevExtreme/pull/16330/commits/fcb3408ec98d8bb1cbf5cac06409c49357272be7